subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/transform/TransformationLoggingIntegrationTest.groovy

```diff
@@ -33,6 +34,7 @@ class TransformationLoggingIntegrationTest extends AbstractConsoleGroupedTaskFun
             include 'util'
             include 'app'
         """
+        enableIncrementalArtifactTransformations(settingsFile)
```

Comment:
Should we run this instead with both incremental enabled and disabled?

Reply:
The reason why I don't run it with both incremental enabled and disabled is that the test already uses a runner - and runners are not composable :(

I opted for testing with incremental transformations, since this will be the default.